client/src/models/results.js

```diff
@@ -508,6 +511,10 @@ const get_result_doc_keys = (doc) => _.chain(result_docs)
   })
   .value();
 
+const current_drr_key = _.chain( get_result_doc_keys('drr') ).sortBy().last().value();
+const current_dp_key = _.chain( get_result_doc_keys('dp') ).sortBy().last().value();
```

Comment:
I think it makes more sense for the keys to come from `get_result_doc_keys` sorted.

Reply:
Good idea.

Jokes on you though, I can't re-request review because I hadn't even requested one yet! Still a good bit more to tackle on this branch I think lots of annoying hard coded doc names in the results count code (hard coded by me...).

Reply:
wait you didn't? shoot, I got too eager

Reply:
Aha